LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/flex-item-transferred-sizes-padding-border-sizing-expected.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="author" title="Sammy Gill" href="mailto:sammy.gill@apple.com" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing-4/#aspect-ratio-size-transfers" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing-3/#min-width" />
+    <link rel="help" href="https://drafts.csswg.org/css-sizing/#box-sizing" />
+    <link rel="help" href="https://bugs.webkit.org/show_bug.cgi?id=243887" />
+    <link rel="match" href="../reference/ref-filled-green-100px-square.xht" />
+  </head>
+    <style>
+      div {
+        width: 100px;
+        height: 100px;
+        background-color: green;
+      }
+    </style>
+  <body>
+    <div></div>
+  </body>
+</html>
```

Comment:
You can probably get rid of the "no newline at eof" message.

Reply:
Fixed